src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -400,20 +408,20 @@ def _setup_fuzzer_and_device(self):
     test_data_dir = os.path.join(fuchsia_resources_dir_plus_build,
                                  self.FUZZER_TEST_DATA_REL_PATH, package,
                                  target)
+
+    # Finally, we set up the Fuzzer object itself, which will run our fuzzer!
     self.fuzzer = Fuzzer(
         self.device, package, target, output=test_data_dir, foreground=True)
 
   def __init__(self, executable_path, default_args=None):
-    qemu_path, qemu_args = setup_qemu_values(initial_setup=False)
-    qemu_process = setup_qemu_instance(qemu_path, qemu_args)
-    self._setup_fuzzer_and_device()
-    self.qemu_instance = run_qemu_instance(qemu_process)
+    self.qemu = None
+    self._setup_qemu_device_and_fuzzer()
```

Comment:
I don't think we should keep running a new qemu instance every time we initialise the LibfuzzerRunner, since we create several in a several run, leading to killing/restarting every time we try do a reproduction round for instance. It's probably better to revert to starting the initial process once the build is setup, but keep the restart logic.

Reply:
chatted offline; refactored so that this logic works cleanly (via starting process in build_manager.py and leaving furhter manipulation in libfuzzer.py)